R/mc4_mthds.R

```diff
@@ -46,6 +46,10 @@
 #'     required for tcplfit2 processing.}
 #'     \item{bidirectional.false}{Limits bidirectional fitting and processes data in positive 
 #'     analysis direction only. Use for gain-of-signal or inverted data.}
+#'     \item{bmad5.onesd16.static}{Replace bmad (baseline median absolute deviation) with 5 and osd
+#'     (one standard deviation of the normalized response for test compound wells (wllt = t) with a
+#'     concentration index (cndx) of 1 or 2) with 16. Typically used for binary data where values
+#'     would otherwise be 0 since non-zero values are required for tcplfit2 processing.}
```

Comment:
Suggestion to reword this section as follows:

Replace baseline median absolute deviation (bmad) with 5 and one standard deviation of the normalized response for test compound wells (wllt = t) with a concentration index (cndx) of 1 or 2 (osd) with 16. Typically used for binary data where values would otherwise be 0; non-zero values are required for tcplfit2 processing.

Reply:
description updated mc4_mthds.R and in mc4_mthds table in invitrodb